transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollServerChannel.java

```diff
@@ -75,12 +75,8 @@ protected Object filterOutboundMessage(Object msg) throws Exception {
 
     @UnstableApi
     @Override
-    protected final void doShutdownOutput(Throwable cause) throws Exception {
-        try {
-            super.doShutdownOutput(cause);
-        } finally {
-            close();
-        }
+    protected final void doShutdownOutput() throws Exception {
+        doClose();
```

Comment:
should we just make this the default method behavior in `AbstractChannel` and the channels which don't want to close can override and do a no-op?

Reply:
@Scottmitch I thought about this but did not do it as I was thinking that this may have "bad effects" for people the implemented their own transport. WDYT ?

Reply:
Hum it is possible but I wonder if its better to explicitly close in this case instead of silently doing nothing. It seems easier to detect this behavior rather than doing nothing and continuing to queue writes. Also if someone has a custom protocol and is calling `shutdownOutput` it seems likely they will expect some action to be taken and not just a noop.

Reply:
good point... let me do this.